.eslintignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
This folder shouldn't be committed because it contains only auto generated code

Reply:
added